src/snovault/tests/test_link.py

```
@@ -26,11 +26,11 @@
 def content(testapp):
     url = '/testing-link-targets-sno/'
     for item in targets:
-        testapp.post_json(url, item, status=201)
+        res = testapp.post_json(url, item, status=201)
 
     url = '/testing-link-sources-sno/'
     for item in sources:
-        testapp.post_json(url, item, status=201)
+        res = testapp.post_json(url, item)
```

This and the above change was for testing, right? Should probably be changed back since I beliee this one will fail silently (and you don't need res?)

---

Yep, definitely. Reverted.